extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -117,6 +125,45 @@ export class Bind {
      */
     this.digestQueuedAfterScan_ = false;
 
+    /**
+     * @const @private {!Array<Node>}
+     */
+    this.dynamicRoots_ = [];
```

Comment:
I can't reproduce the issue now. I think it looked like it was calling this with one of the added children as the target due to a problem in `test-bind-impl`. Done.